packages/integration-tests/fixtures/esbuild-inject-compat/input/inject.ts

```diff
@@ -0,0 +1,3 @@
+export const process = { cwd: () => "", env: {} };
```

Comment:
Is the idea that this _overwrites_ `process.env.FOO`? Where would that be set?

Reply:
I could make the test a bit clearer actually...

Reply:
Adjusted here [d32130e](https://github.com/getsentry/sentry-javascript-bundler-plugins/pull/416/commits/d32130e5ad6bac1f6b64d7ae6b681eb40cadefb8)